### PR TITLE
Rebrand to syndesis.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,26 +49,26 @@
...
   </scm>
 
   <distributionManagement>
     <repository>
-      <id>bintray-redhat-ipaas-maven</id>
-      <name>redhat-ipaas-maven</name>
-      <url>https://api.bintray.com/maven/redhat-ipaas/maven/connectors/;publish=1</url>
+      <id>bintray-syndesis-maven</id>
+      <name>syndesis-maven</name>
+      <url>https://api.bintray.com/maven/syndesisio/maven/connectors/;publish=1</url>
```
Comment (jimmidyson) [inline pom.xml]: We don't need these any more - we can distribute straight to Maven Central now. @rhuss Did you have the details for io.syndesis gid?

File: pom.xml [hunk 2]
```diff
@@ -49,26 +49,26 @@
     central -->
   <developers>
     <developer>
-      <id>ipaas</id>
-      <name>iPaaS Development Team</name>
+      <id>syndesis</id>
+      <name>Syndesis Development Team</name>
       <organization>redhat</organization>
       <organizationUrl>http://redhat.com/</organizationUrl>
```
Comment (jimmidyson) [inline pom.xml]: Probably remove these organization bits as this is upstream project.

File: pom.xml (hunk 1 above)
Comment (rhuss) [reply pom.xml]: its available now: https://issues.sonatype.org/browse/OSSRH-30988

**Comment by iocanel** [comment]
retest this please

**Comment by syndesisci** [comment]
retest this please

**Comment by iocanel** [comment]
retest this please

**Pull request merged**